package/gluon-mesh-batman-adv-core/files/lib/gluon/upgrade/310-gluon-mesh-batman-adv-core-mesh

```diff
@@ -20,22 +21,25 @@ uci:commit('batman-adv')
 
 
 if not uci:get('network', 'client') then
-  local ifname
+  uci:section('network', 'interface', 'client',
+    {
+      type = 'bridge',
+      proto = 'dhcpv6',
+      reqprefix = 'no',
+    }
+  )
+
+  uci:add_to_set('network', 'client', 'ifname', 'bat0')
```

Comment:
I have prepared a package containing all useful helpers named simple-uci (https://github.com/freifunk-gluon/packages/commit/9e81355843f7d0f7b59ff2f1fdb8980ad6bc31c2). It's a drop-in for luci.model.cbi using the same API (excluding any luci specific calls).